custom/icds_reports/migrations/0151_daily_attendance_state.py

```diff
@@ -0,0 +1,29 @@
+from django.db import migrations
+
+from custom.icds_reports.utils.migrations import create_index_migration
+
+
+migrator = RawSQLMigration(('custom', 'icds_reports', 'migrations', 'sql_templates', 'database_views'))
+
+class Migration(migrations.Migration):
+
+    table_name = "daily_attendance"
+    index_name = "ix_daily_attendance_month_state_id"
+    columns = ['month', 'state_id']
+
+    create_index_sql, drop_index_sql = create_index_migration(table_name, index_name, columns)
+
+    atomic = False
+
+    dependencies = [
+        ('icds_reports', '0150_update_agg_awc_monthly_view'),
+    ]
+
+    operations = [
+        migrations.RunSQL("ALTER TABLE daily_attendance ADD COLUMN state_id text"),
+        migrations.RunSQL(
+            sql=create_index_sql,
+            reverse_sql=drop_index_sql,
+        ),
+        migrator.get_migration('daily_attendance.sql')
```

Comment:
@calellowitz  I am guessing this is because you had to recreate the view in order to prevent  `ERROR:  column 124 of relation "<table_name>" does not exist`

Reply:
yes, that